src/seedu/addressbook/data/AddressBook.java

```diff
@@ -128,4 +128,6 @@ public boolean equals(Object other) {
                         && this.allPersons.equals(((AddressBook) other).allPersons)
                         && this.allTags.equals(((AddressBook) other).allTags));
     }
+
+    public UniquePersonList getUniquePersonList(){ return allPersons;}
```

Comment:
1. Please check the coding standards on how to write methods containing single line of code.
2. Need a space between `) {`